### ISSUE TITLE
A shame....a damn shame

### ISSUE THREAD

**Issue by bodln** [dejanaksovic/angular#1]
-First thing first is that the invalid input error message once shown never goes away. It's mere existence galls me for it infers i did something wrong when i most certainly did not and even if i did it was definitely intentional for i am not one known for his flaws.

-Second is that godawful delete button which upon it's invocation for a single element vanquishes all of it's peers to the great unknown, of course the great unknown in this case being why does it do that in the first place.

-And last but also the least is upon the caress and gentle press of my cursor on the so called "snimi" button the Hogwarts logo mysteriously disappears and becomes a small off centered icon.

In conclusion this repository is an affront to all it's betters (by which i mean everyone ever) and should have never even sullied the eyes of the public.

Best wishes and most sincerest goodbyes by your's truly,
fuck you.

**Comment by D3kky** [comment]
1. The error message. Bug fixed, main problem was that I would switch prop that would decided if it was shown manually. Instead of having a "get" prop which would do it dynamically every time it was requested.

2. Delete. Bug fixed. The problem was that courses had no keys (i.e. id) which would decided which course should be deleted, so I used course values, such as name, which means that all courses with requested deleting name would be sent to the shadow realm. With the last update, where I introduced models, which have individual ids it was an easy fix. 

3. The logo. It is supposed to take in the image url, at which point the application would change the image on the screen to image provided through url. Bug not fixed. I hope to do it in short notice, probably trying to check if the url provided actually leads to an actual image.

All in all thank you for kind words and guidance. I hope you will review these changes and we can go over them, as well as the new features added.

Sincerely,
Much better developer

Ps.
If you are so keen on advancing repository, make a pull request and I will consider merging it.

**Comment by bodln** [comment]
Sincerely, i don't care how you did it just do it you corporate fucking
slave
